recipes/tune-sklearn/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "tune-sklearn" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # NOTE: pypi doesnt contain a source package for it. See
+  # https://github.com/ray-project/tune-sklearn/issues/159
+  url: https://github.com/ray-project/tune-sklearn/archive/v{{ version }}.tar.gz
+  sha256: 436cc5fa37d9cb596ce8a841b1c20120564d012114ebefd024aebdac753ce5f3
+
+build:
+  # noarch: python
+  number: 0
+  skip: true  # [osx]
```

Comment:
Better to remove this and make this `noarch:python`. The lack of `ray` on osx isn't a concern here. Also, if `ray` ever works on osx, this package will magically be installable.

Reply:
Yes, that's what I need first but the osx obviously fails. I guess it's fine?

Reply:
reverted back to noarch.

Reply:
Yeah, that's fine. Being noarch, it won't affect package builds in the feedstock (we use Linux to build noarch packages).  Mac users just don't get to install it.